pypeit/scripts/ql_keck_mosfire.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE file for Copyright information
+#
+# -*- coding: utf-8 -*-
+"""
+This script runs PypeIt on a pair of MOSFIRE images (A-B)
+"""
+import argparse
+
+from pypeit import msgs
+
+import warnings
+
+def parser(options=None):
+
+    parser = argparse.ArgumentParser(description='Script to run PypeIt on MOSFIRE in A-B mode')
+    parser.add_argument('full_rawpath', type=str, help='Full path to the raw files')
+    parser.add_argument('fileA', type=str, help='A frame')
+    parser.add_argument('fileB', type=str, help='B frame')
+    parser.add_argument('flat', type=str, help='Flat frame filename for tracing the slits')
```

Comment:
Should this be traceflat, not flat to make it more clear what we use this for. We don't want to flat field or illum correct for quicklook. Give them an option to pass a slit trace object already?

Looking forward. We need an init_quicklook script that scours through their data directory, finds a flat for each mask and simply creates the MasterSlits after their afternoon calibrations.